Assets/MixedRealityToolkit.Extensions/Socketer/Scripts/TCPConnectionManager.cs

```diff
@@ -25,15 +32,14 @@ public class TCPConnectionManager : MonoBehaviour
         /// </summary>
         public event Action<IncomingMessage> OnReceive;
 
-        private readonly ConcurrentQueue<SocketEndpoint> newConnections = new ConcurrentQueue<SocketEndpoint>();
-        private readonly ConcurrentQueue<SocketEndpoint> oldConnections = new ConcurrentQueue<SocketEndpoint>();
-        private readonly ConcurrentDictionary<int, SocketEndpoint> serverConnections = new ConcurrentDictionary<int, SocketEndpoint>();
+        private ConcurrentQueue<SocketEndpoint> newConnections;
```

Comment:
Why not keep these auto initialized and readonly?

Reply:
These constructors weren't seeming to work correctly for the concurrent data structures when moving to a monobehaviour. Having now removed the constructor call from the test script, I'll give this another shot.

Reply:
readonly fields can only be initialized in their declarations or in the constructor. by making this a monobehavior (and initializing them in awake) they cannot be readonly fields.

to preserve the previous behavior, consider initializing in the declarations (like previously) and not in Awake.